chartsxblock/static/js/src/chartsxblock-studio.js

```diff
@@ -183,31 +198,55 @@ function ChartTable(targetDiv, inputJSON){
 
     this.addRow = function() {
         var rowSelector = $('<tr>');
-        this.div.find("tbody").append(rowSelector);
         for (var i = 0; i < this.columns; i++) {
             rowSelector.append($('<td>').makeEditable());
         }
+        rowSelector.append($('<td class="actions"><button class="deleteRow fa fa-times" aria-hidden="true"></button></td>'));
+        this.div.find("tbody").append(rowSelector);
         this.rows++;
     };
 
-    this.removeRow = function() {
+    this.removeRow = function(row=null) {
+        if(this.rows <2) throw {name: "Dimensions Error", message: "You cannot have 0 rows."};
+        if(row) {
+            row.remove();
+        }
+        else {
+            this.removeLastRow();
```

Comment:
Zašto imati funkciju koja se jedino poziva iz druge funkcije? Jednostavnije je prebaciti kod ovdje, ne? :smile: